Replace service calls for redis/valkey and grommunio-antispam which is a relabeled rspamd
--- a/data/config.yaml.in
+++ b/data/config.yaml.in
@@ -14,7 +14,6 @@
     local: 'http://localhost:8080/statusmonitor'
   dashboard:
     services:
-      - unit: grommunio-antispam.service
       - unit: gromox-delivery.service
       - unit: gromox-event.service
       - unit: gromox-http.service
@@ -27,7 +26,8 @@
       - unit: nginx.service
       - unit: '@PHP_FPM_SERVICE@'
       - unit: postfix.service
-      - unit: redis@grommunio.service
+      - unit: valkey@grommunio.service
+      - unit: rspamd.service
 mconf:
   ldapPath: '/etc/gromox/ldap_adaptor.cfg'
   authmgrPath: '/etc/gromox/authmgr.cfg'
@@ -37,7 +37,7 @@
   Admin API:
     source: 'grommunio-admin-api.service'
   Antispam:
-    source: 'grommunio-antispam.service'
+    source: 'rspamd.service'
   Gromox http:
     source: 'gromox-http.service'
   Gromox smtp:
